app/screens/onboarding/footer_buttons.tsx

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Pressable, useWindowDimensions, View} from 'react-native';
+import Animated, {Extrapolate, interpolate, useAnimatedStyle} from 'react-native-reanimated';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {ONBOARDING_CONTENT_MAX_WIDTH} from '@screens/onboarding/slide';
+import {buttonBackgroundStyle, buttonTextStyle} from '@utils/buttonStyles';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    theme: Theme;
+    lastSlideIndex: number;
+    nextSlideHandler: () => void;
+    signInHandler: () => void;
+    scrollX: Animated.SharedValue<number>;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    button: {
+        marginTop: 5,
+    },
+    rowIcon: {
+        color: theme.buttonColor,
+        fontSize: 12,
+        marginLeft: 5,
+        marginTop: 4.5,
+    },
+    nextButtonText: {
+        flexDirection: 'row',
+        position: 'absolute',
+    },
+    signInButtonText: {
+        flexDirection: 'row',
+    },
+    footerButtonsContainer: {
+        flexDirection: 'column',
+        height: 120,
+        marginTop: 25,
+        marginBottom: 15,
+        width: '100%',
+        alignItems: 'center',
+    },
+}));
+
+const AnimatedButton = Animated.createAnimatedComponent(Pressable);
```

Comment:
is there

Reply:
The standard on our app is the react-native-button. The main reason to use it is that behaves like a button both in iOS and Android (they have different animations when clicked, mainly). So, by default, we should be using react-native-button (even though, this is not true in the code, and I would like to refactor that eventually).

Therefore, I wanted to make sure if you have tried using react-native-button and were forced to use something different, or just didn't know about react-native-button.

Reply:
ok, so, there was no special reason for me to use one over the other. Is it relevant for this case to replace the Pressable with the Button?

Reply:
I think it is relevant to use `Button` here, since it is indeed a Button. Pressable, if I am not mistaken, it is targetted for more general scenarios where you need extra configuration, which I think it is not the case. Nevertheless, what do you think @enahum ?

Reply:
Pressable is totally fine, but if we can standarize it to use button, that'll be great.. obviously it has to be animated